az-digital/az_quickstart
# Closes #2527: Add badge view mode, update existing badge view mode to "linked badge" and setting terms up to use them.

## Description
Adding new view mode for taxonomy (Badge no link)
Adding new displays for `az_publication_categories`

Rename badge to linked badge

Possible todos:

- [ ] Add both displays to all vocabularies that have badge
- [ ] Add new display to az_publication, reference with tags
- [ ] Add new display to az_publication, reference with linked tags.

## Related issues
Closes #2527 

## How to test general term integration
Log in as admin
Navigate to /admin/structure/taxonomy/manage/az_page_categories/overview/display (Or any other taxonomy)
<img width="1659" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/1f3b6657-1145-4b81-9e41-95200b0ba51c">
You should now be able to enable the badge display (not a link)
Also you should see the new name of the existing badge display (linked badge)

## How to test publication term integration
Navigate to /admin/structure/taxonomy/manage/az_publication_categories/overview/display/az_badge
<img width="1662" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/8b40a49a-c7c2-46ca-9c71-ffc5ca9b40c8">


Navigate to /admin/structure/taxonomy/manage/az_publication_categories/overview/display/az_badge_no_link
<img width="1673" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/7984b16c-591a-42de-bf46-02e49bc33dba">


To enable this on an entity reference field in a view display for a fieldable entity, here are the settings you need.
<img width="1635" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/8c97265d-b425-4044-b28d-7804efb27c8f">

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
   - [] Risky or disruptive cleanup to comply with coding standards
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
